cli/internal/cmd/workspace/retl-source.go

```diff
@@ -0,0 +1,65 @@
+package workspace
+
+import (
+	"fmt"
+
+	"github.com/rudderlabs/rudder-iac/cli/internal/app"
+	"github.com/rudderlabs/rudder-iac/cli/internal/cmd/telemetry"
+	"github.com/rudderlabs/rudder-iac/cli/internal/lister"
+	"github.com/rudderlabs/rudder-iac/cli/internal/providers/retl"
+	"github.com/rudderlabs/rudder-iac/cli/internal/providers/retl/sqlmodel"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdRetlSource() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "retl-source",
```

Comment:
Let's call this `retl-sources` (plural) to be inline with `rudder-cli workspace accounts`